gotopkg.go

```diff
@@ -1,204 +0,0 @@
-package main
-
-import (
-	"flag"
-	"fmt"
-	"github.com/groob/plist"
-	"io"
-	"io/ioutil"
-	"log"
-	"os"
-	"os/exec"
-	"strings"
-	"syscall"
-	"time"
-)
-
-const RECIPE_FAILED_CODE = 70
-
-type AutopkgRunReport struct {
-	Failures []struct {
-		Recipe    string `plist:"recipe"`
-		Message   string `plist:"message"`
-		Traceback string `plist:"traceback"`
-	} `plist:"failures"`
-	SummaryResults struct {
-		URLDownloaderSummaryResult struct {
-			DataRows []struct {
-				DownloadPath string `plist:"download_path"`
-			} `plist:"data_rows"`
-			Header      []string `plist:"header"`
-			SummaryText string   `plist:"summary_text"`
-		} `plist:"url_downloader_summary_result"`
-		MunkiImporterSummaryResult struct {
-			DataRows []struct {
-				PkgRepoPath string `plist:"pkg_repo_path"`
-				Catalogs    string `plist:"catalogs"`
-				Version     string `plist:"version"`
-				PkginfoPath string `plist:"pkginfo_path"`
-				Name        string `plist:"name"`
-			} `plist:"data_rows"`
-			Header      []string `plist:"header"`
-			SummaryText string   `plist:"summary_text"`
-		} `plist:"munki_importer_summary_result"`
-	} `plist:"summary_results"`
-}
-
-func (rpt *AutopkgRunReport) DecodePlist(r io.Reader) error {
-	err := plist.NewDecoder(r).Decode(&rpt)
-	if err != nil {
-		return fmt.Errorf("error decoding report from reader:", err)
-	}
-
-	return nil
-}
-
-var autopkg string
-
-func main() {
-	flagAutopkg := flag.String("autopkg", "autopkg", "path to autopkg")
-	flag.Parse()
-
-	path, err := exec.LookPath(*flagAutopkg)
-	if err != nil {
-		log.Fatalf("error finding autopkg binary: %s", err)
-	}
-
-	autopkg = path
-	log.Printf("using autopkg: %s\n", autopkg)
-
-	if flag.NArg() < 1 {
```

Comment:
Not instead. you still need to exit, but the `flag.Usage()` will print out the flags and any help text you attach.